assets/eip-1/useful-submissions.md

```diff
@@ -0,0 +1,55 @@
+| Food nutrition labels ([US](https://www.fda.gov/food/nutrition-education-resources-materials/new-nutrition-facts-label), [EU](https://food.ec.europa.eu/safety/labelling-and-nutrition/food-information-consumers-legislation/nutrition-labelling_en), [Shanghai, CN](https://www.shanghai.gov.cn/nw42885/index.html)) | Commercial preparers of packaged food                     | Government and commercial food inspectors, people that eat food |
+| People drive on the left side of the road (in some countries) | Drivers, car manufacturers, traffic signage               | Other drivers on the road, pedestrians                       |
+
+Let's look a little closer at things that are and are not standardized:
+
+| Standard                                  | Origin of standard                                           |
+| ----------------------------------------- | ------------------------------------------------------------ |
+| Food labeled as "USDA organic" in USA     | [USDA enforces](https://www.ams.usda.gov/rules-regulations/organic) usage of this word |
+| Cherios box claims its oats are "toasted" | There is no applicable definition or enforcement of what "toasting" is, your recourse is may only include a civil action |
+
+We have looked at a few example of things that have been standardized, different kinds of standards. And you might think, anything can be rewritten as a standard if you really wanted to. And you're right. Here are some contrived examples:
+
+* When I walk down the street, I don't skip. That's **a walking standard**.
+* My smart contract allows to mint pictures of tables using a `mintTable` function. This is **a minting standard for NFTs**.
+* I wrote some software and created test cases. All of those test cases are **a software standard**.
+
+Some of these behaviors might be important. Maybe you don't like skipping, you can adopt that behavior. But as a standard these are useless, I hope you understand. But let's articulate why.
+
+<mark>A standard only deserves to be written if multiple people adhere to it and people depend on its surface area.</mark>
+
+FYI, in the United States, the post office has their own police. Watch out for that.
+
+✅ If you routinely use postcard stamps to mail letters that are not postcard size you will go to jail. Multiple people mail postcards and depend on not going to jail... this deserves to be standardized.
+
+❌ Your smart contract uses a specific way to track inclusion of addresses in a set. Nobody cares or is affected by which specific way you track inclusion of addresses in a set. This does not deserve to be standardized.
+
+An easy, low-effort, low-quality way to find a bunch of things to standardize is to find a popular repository of smart contracts, and then write every one of them up with a short rationale. This is fun for practice, but please do not publish these as it will be a waste of other people's time to read. Nobody depends on these details.
```

Comment:
```suggestion
An easy way to find a bunch of things to standardize is to find a popular repository of smart contracts, and then write every one of them up with a short rationale. This is fun as practice, but unless you have found a strong motivation, please do not submit these.
```
Although I generally agree with the sentiment, those last few sentences are too harsh, even by my standards.